src/kirby/components/chip/chip.component.scss

```
@@ -14,34 +14,34 @@ $chip-height: 30px;
   margin-right: 5px;
   cursor: pointer;
   background-color: get-color('white');
-  color: get-color('black');
+  color: get-text-color('black');
   user-select: none;
 
   &:hover,
   &.is-selected {
     background-color: get-color('black');
-    color: get-color('white');
+    color: get-text-color('white');
   }
 
   :host-context(.default),
   :host-context(.kirby-color-brightness-white) {
     background-color: get-color('white');
-    color: get-color('black');
+    color: get-text-color('black');
```

Use contrast instead;
```suggestion
  color: get-color('white-contrast');
```